src/epics/externalLayers.js

```diff
@@ -5,7 +5,7 @@ import 'rxjs/add/operator/mergeMap';
 import 'rxjs/add/operator/catch';
 import 'rxjs/add/operator/concatAll';
 import 'rxjs/add/operator/filter';
-import { Observable } from 'rxjs/Observable';
+import { Observable } from 'rxjs';
```

Comment:
Have you checked to see if this change affects the bundle size?

Reply:
No, hopefully we'll be able to remove the rx.js dependency for 2.35.